### DISCUSSION TITLE
HD Belzebub mod

### DISCUSSION THREAD

**Discussion by Zexxo5** [diasurgical/DevilutionX#4071] [Q&A]
Hi all. I'm new here and new in the mac os world too. How can I install the Belzebub mod?
here is: https://mod.diablo.noktis.pl/download

**Comment by AJenbo** [comment]
Belzebub is a different engine unrelated to DevilutionX (and also Diablo). I don't think Belzebub supports Mac (or anything but Windows for that matter).

**Reply by Zexxo5** [reply]
thanks for the fast reply :) what mods can I use?

**Reply by AJenbo** [reply]
Nearly all mods work by replacing the Windows executable, since the game has no support for mods. There are a few [mods based on DevilutionX](https://github.com/diasurgical/devilutionX/wiki/Mods-and-related-projects), but since it also doesn't have proper mod support (yet), it is up to the author of these mods to maintain support for various platforms. There are some efforts happening to port some old mods to DevilutionX as well.
